exercises/APE7/0702.tex

```diff
@@ -0,0 +1,53 @@
+\paragraph{a.} Let's first define what a belief vector is : for $\pi \in  \times_{s \in S} \Delta Y_s $, $i \in N$, $s \in S_i$, and $x \in Y_s$, $\pi_s(x)$ is the probability assigned by player $i$ to be at node $x$ given the fact that $i$ knows his information state $s$. ($X$ the nodes, $Y_s\in X$ is the set of nodes that have information $s$). \\
```

Comment:
Don't use `\\` outside a `{tabular}`. Leave a blank line if you want to create a new paragraph